packages/mdc-textfield/_mixins.scss

```diff
@@ -405,7 +407,9 @@
 
     top: $label-offset-y;
     bottom: auto;
-    padding: 8px 16px;
+    margin-top: 2px;
+    margin-left: 8px;
+    padding: 8px;
```

Comment:
Is this a new change? Or did you move existing logic into here?

Reply:
Unrelated. This was my merge with master.